Assets/VRTK/Scripts/Helper/VRTK_EventSystemVRInput.cs

```diff
@@ -153,13 +154,70 @@ private void Hover(VRTK_UIPointer pointer, List<RaycastResult> results)
 
         private void Click(VRTK_UIPointer pointer, List<RaycastResult> results)
         {
-            pointer.pointerEventData.eligibleForClick = pointer.controller.uiClickPressed;
+            switch (pointer.clickMethod)
+            {
+                case VRTK_UIPointer.ClickMethods.Click_On_Button_Up:
+                    ClickOnUp(pointer, results);
```

Comment:
is this needed if this is implemented:

https://github.com/thestonefox/VRTK/issues/685